inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -111,3 +113,10 @@ def is_record_relevant(obj, *args, **kwargs):
                 len(core_keywords) == 0:
             return False
     return True
+
+
+def is_experimental_paper(obj, eng):
+    """Is the record an experimental paper."""
+    categories = get_value(obj.data, "arxiv_eprints.categories", []) + \
+        get_value(obj.data, "subject_terms.term", [])
+    return "hep-ex" in categories or "Experiment-HEP" in categories
```

Comment:
Good point! The 2 fields are definitively correlated. I wonder which one comes from arXiv?